docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
+:getting-started-url-connectors: connectors/getting-started-connectors/README.adoc
+:getting-started-rhoas-cli-url-connectors: connectors/rhoas-cli-getting-started-connectors/README.adoc
+
+//OpenShift API Designer
+:product-long-api-designer: OpenShift API Designer
+:product-api-designer: API Designer
+:product-version-api-designer: 1
+:service-url-api-designer: https://console.redhat.com/application-services/api-designer/
+:getting-started-url-api-designer: api-designer/getting-started-api-designer/README.adoc
+
+//OpenShift API Management
+:product-long-api-management: OpenShift API Management
+:product-api-management: API Management
+:product-version-api-management: 1
+:service-url-api-management: https://console.redhat.com/application-services/api-management/
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-using-iac-tools"]
+= Automating deployment and management of Kafka instances in {product-long-kafka}
+ifdef::context[:parent-context: {context}]
+:context: using-iac-tools
+
+[role="_abstract"]
+As a system administrator of applications and services, you can use Infrastructure as Code (IaC) tools to automate deployment and management of architectures.
+
+This guide describes how to use different IaC tools to provision and manage Kafka instances in {product-long-kafka}. To interact with Kafka instances, the IaC tools use two REST APIs that are available in {product-kafka} - a _management_ API and an _instance_ API.
+
+ifndef::community[]
+For more information about these APIs, see https://access.redhat.com/documentation/en-us/red_hat_openshift_application_services/1/guide/2409253a-45ee-470e-bdc9-5db4bfcf9d0f.
+endif::[]
+
+For more information on creating and setting up Kafka instances, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
+
+
+//Additional line break to resolve mod docs generation error
+
+[id="con-ansible_{context}"]
+== Ansible
+
+[role="_abstract"]
+https://www.ansible.com/overview/how-ansible-works[Ansible^] is an open source configuration management and automation tool.
+
+Ansible uses modules to execute tasks. The output of one module can be the input for the next one or they can be independent of each other.
+
+You can combine one or more Ansible modules to make a play. Combine two or more plays to create an Ansible Playbook, a list of sequential tasks (written in YAML) that automatically executes when you run the `ansible-playbook` command. You can use an Ansible Playbook to describe the desired state of your infrastructure and then have Ansible provision it.
+
+Ansible roles are a special kind of playbook that is fully self-contained and portable with the tasks, variables, configuration templates, and other supporting files that are needed to complete your chosen tasks. Multiple roles can exist inside a collection allowing easy sharing of content through Automation Hub and https://galaxy.ansible.com[Ansible Galaxy^].
```

Comment:
Something here doesn't read quite right: "Ansible roles **are**.....kind of playbook that **is**....

Can we solve this by saying "**An** Ansible role **is**..."?

Reply:
"Multiple roles can exist in a collection,..." (introduce comma)